internal/services/engines/swift/rules.go

```diff
@@ -190,7 +186,7 @@ func NewMD6Collision() *text.Rule {
 			Severity:    severities.Medium.ToString(),
 			Confidence:  confidence.Low.ToString(),
 		},
-		Type: text.OrMatch,
+		Type: text.Regular,
```

Comment:
Why is changed to Regular?

Reply:
I believe it is due to the future deprecation of OrMatch, but I believe that maybe it should be in a future pr specific for this change.

Reply:
yes, I changed all to Regular

Reply:
I made a mistake here, sorry. We are going to depreciated the Regular not the OrMatch. This change is kinda pointless. Sorry.

Reply:
Oh... Okay. I'll change all to OrMatch

Reply:
done! @nathanmartinszup